build-debezium.sh

```diff
@@ -2,6 +2,24 @@
 
 set -eo pipefail
 
+# For testing purposes, you can set your own docker hub username
+#    (dont forget to 'docker login' before running the script)
+# Also you can specify an own second registry as a replacement
+# for the 'official' quay.io
+#    For testing you can could use a local docker registry:
+#      docker run -d -p 5000:5000 --restart=always --name registry registry:2
+#    and then set DEBEZIUM_QUAY_IO=localhost:5000
+# (and don't forget to docker login)
+
+if [ -z "${DEBEZIUM_DOCKER_NAME}" ]; then
+  DEBEZIUM_DOCKER_NAME=debezium
+fi;
+
+if [ -z "${DEBEZIUM_QUAY_IO}" ]; then
+  DEBEZIUM_QUAY_IO=quay.io
```

Comment:
@nilshartmann  Am I missing something? This does not look to be used anywhere else in the context of debezium build.  For testing / dry run it might be useful to specify to use a local registry as well (buildx currently doesn't support builds from local storage for multiarch build and it needs the image to be pushed). 


This also leads me to a opinion that the semantics of `DEBEZIUM_DOCKER_NAME` and `DEBEZIUM_QUAY_IO` is not clear.